qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/Shape2D.h

```diff
@@ -6,6 +6,7 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 #pragma once
 
+#include "MantidAPI/DllConfig.h"
```

Comment:
Is this needed?

Reply:
It is needed for UNUSED_ARG (which is useful for the override of setBoundingRect, where nothing happens)